modules/ballerina-builtin/src/main/ballerina/ballerina/util/natives.bal

```diff
@@ -46,3 +46,9 @@ public native function getHmacFromBase64 (string baseString, string keyString, s
 @Param { value:"algorithm: The hashing algorithm to be used" }
 @Return { value:"The hashed string" }
 public native function getHmac (string baseString, string keyString, string algorithm) (string);
+
+@Description { value:"Verify given JWT token according to the given key"}
+@Param { value:"jwToken: JSON Web Token" }
+@Param { value:"key: (if JWT encrypt with RSA algorithm this is the public key, if JWT hashed this is the secret key)" }
+@Return { value:"boolean: Returns JWT is valid or not" }
+public native function verifyJwt (string jwToken, string key) (boolean);
```

Comment:
Also when you add a new Functionality, make sure you have added relevant test cases as well.